src/mame/rm/rm380z.h

```diff
@@ -79,6 +83,8 @@ class rm380z_state : public driver_device
 		uint8_t m_scroll_reg = 0;
 	};
 
+	enum hrg_display_mode { RM380Z_HRG_MODE_NONE, RM380Z_HRG_MODE_HIGH, RM380Z_HRG_MODE_MEDIUM_0, RM380Z_HRG_MODE_MEDIUM_1 };
```

Comment:
Eventually this system should be updated to support saved states.  In preparation for that, please give all enums used for state explicit storage classes.  Also consider using a scoped enum (`enum class`) if it’s a closed set of values.

Reply:
I wasn't sure what you mean by explicit storage classes, but I've made it a scoped enum with explicit values.  Is it correct now?

Reply:
He means that you can specify an actual base type for an enum, like:

enum foo: uint32_t
{
    blah=0x12345678
}

Reply:
ah, thank you @rb6502.  I didn't even know you could do that.  I always think of enum as being an int, but good to know.  I have updated it again and am just about to push the commit.